src/pages/verify/guides/privatekey-jwt.mdx

```diff
@@ -0,0 +1,160 @@
+It relies on using JWT assertions signed with asymmetric key pairs for client authentication.
+
+<Highlight icon="file-lines">
+
+Private Key JWT Authentication (`private_key_jwt`) offers a [more secure alternative](/verify/guides/privatekey-jwt/#advantages-of-private-key-jwt-authentication) to symmetric credentials (`client_secret` authentication options)
+for client authentication in [traditional server-based web applications](/verify/getting-started/oidc-intro/#using-openid-connect-to-integrate-with-criipto-verify).
+
+</Highlight>
+
+You can configure your [Criitp Verify Application](https://docs.criipto.com/verify/getting-started/basics/#applications) to use `private_key_jwt` for client authentication.
+
+## Authenticate with Private Key JWT
+
+To use `private_key_jwt` for client authentication, follow these three steps:
+
+1. Generate an RSA key pair and a JSON Web Key Set(JWKS).
+
+2. Build the client assertion.
+
+3. Use the client assertion to authenticate your application against Criipto Verify.
+
+You can use one of the recommended libraries for working with JWTs (the list is available at [jwt.io](https://jwt.io/libraries)), or leverage Criipto's [criipto-oidc-js](https://github.com/criipto/criipto-oidc-js) library. We strongly recommend utilizing an established tooling or a third-party library rather than building your own solution.
+
+### Generate an RSA key pair and JWKS
+
+Generate an RSA key pair and a JWKS pointing to the public key. You'll need to [add the JWKS to your Application settings](/verify/guides/privatekey-jwt/#configure-private-key-jwt-authentication) from the Dashboard later on.
+
+The code sample below uses [jose library](https://github.com/panva/jose) to generate a key pair and create a JWKS.
+
+```javascript
+import * as jose from 'jose';
+
+async function generateKeys() {
+  try {
+    // Generate RSA key pair (public and private keys)
+    const { publicKey, privateKey } = await jose.generateKeyPair('RS256');
+    // Export the public key to JWK format
+    const jwk = await jose.exportJWK(publicKey);
+    const jwks = { keys: [jwk] };
+    // Log the JWKS (public key in JWK Set format)
+    console.log('JWK Set (Public Key):', JSON.stringify(jwks, null, 2));
+    // Your private key here. Ensure to store your private key in a secure environment.
+    console.log('Private Key:', privateKey);
+    return { publicKey, privateKey, jwks };
+  } catch (error) {
+    console.error(error);
+    throw error;
+  }
+}
+```
+
+### Build the client assertion
+
+The client assertion is a JWT signed by the private key of the [RSA key pair](/verify/guides/privatekey-jwt/#generate-an-rsa-key-pair-and-jwks) generated earlier. It should contain the following claims:
+
+`alg`: The algorithm used to sign the assertion. Criipto Verify supports RSA256.
```

Comment:
`alg` should be `RS256`.
I suppose we also support other asymmetric algorithms? @kasperhj @sgryt

Reply:
@mickhansen This is left to the MS library to handle and with will probably suck and blow all kinds of algos.
The private key jwt client auth specs only requires an implementation RS256 which is consistent with what we expose in the OIDC metadata.